pages/index.js

```diff
@@ -6,12 +6,15 @@ import transactionChartData from '../mock/transaction-graph-data.json';
 import TransactionList from '../components/transaction-details';
 import coinsData from '../mock/coins.json';
 import CoinsStatus from '../components/coins-status';
-import PersonDetail from '../components/user-info';
 import { Footer } from '../components/footer';
 import styles from '../styles/Home.module.css';
 import TransactionChart from '../components/transaction-chart';
+import OperationModal from '@components/operationModal';
+import { useState } from 'react';
```

Comment:
Please declare this on first line.
If you are importing anything from react, it should be declared right at the top

Reply:
Yes for code readability imports should be mentioned in this order -> external, internal components & libs , style sheets

Reply:
Oh, I will keep this in mind from now onwards.... Thanks guys for your time

Reply:
@sumitd94 changes are added